packages/website-eslint/types/index.d.ts

```diff
@@ -1,13 +0,0 @@
-import type { TSESLint } from '@typescript-eslint/utils';
-
-import { analyze } from '@typescript-eslint/scope-manager/dist/analyze';
-import { astConverter } from '@typescript-eslint/typescript-estree/dist/ast-converter';
-import { getScriptKind } from '@typescript-eslint/typescript-estree/dist/create-program/getScriptKind';
-
-export interface LintUtils {
```

Comment:
are you sure that this is correct?

Reply:
Yes - I moved this to the `website` package.
It wasn't ever actually used in this package, nor did it represent anything that was returned/exported from `website-eslint`.

This also means it's properly covered by typechecks.